translations/base-it.yaml

```diff
@@ -468,125 +469,124 @@ buildings:
+                che può essere usato per controllare, per esempio, un filtro.
     logic_gate:
         default:
-            name: AND Gate
-            description: Emits a boolean "1" if both inputs are truthy. (Truthy means shape,
-                color or boolean "1")
+            name: Porta AND
+            description: Emette un "1" booleano se entrambi gli ingressi sono veri. (Vero significa forma,
+                colore o "1" boolean)
         not:
-            name: NOT Gate
-            description: Emits a boolean "1" if the input is not truthy. (Truthy means
-                shape, color or boolean "1")
+            name: Porta NOT
+            description: Emette un "1" booleano se l'ingresso è falso. (Vero significa forma,
+                colore o "1" booleano)
         xor:
-            name: XOR Gate
-            description: Emits a boolean "1" if one of the inputs is truthy, but not both.
-                (Truthy means shape, color or boolean "1")
+            name: Porta XOR
+            description: Emette un "1" booleano se uno degli ingressi è vero, ma non se lo sono entrambi.
+                (Vero significa forma, colore o "1" booleano)
         or:
-            name: OR Gate
-            description: Emits a boolean "1" if one of the inputs is truthy. (Truthy means
-                shape, color or boolean "1")
+            name: Porta OR
+            description: Emette un "1" booleano se uno degli ingressi è vero. (Vero significa forma,
+                colore o "1" booleano)
     transistor:
         default:
             name: Transistor
-            description: Forwards the bottom input if the side input is truthy (a shape,
-                color or "1").
+            description: Inoltra il segnale dall'ingresso inferiore se l'ingresso laterale è vero
+                (una forma, un colore o "1").
         mirrored:
             name: Transistor
-            description: Forwards the bottom input if the side input is truthy (a shape,
-                color or "1").
+            description: Inoltra il segnale dall'ingresso inferiore se l'ingresso laterale è vero
+                (una forma, un colore o "1").
     filter:
         default:
-            name: Filter
-            description: Connect a signal to route all matching items to the top and the
-                remaining to the right. Can be controlled with boolean signals
-                too.
+            name: Filtro
+            description: Collega un segnale per mandare tutti gli oggetti corrispondenti verso l'alto e
+                i restanti verso destra. Può essere controllato anche con segnali booleani.
     display:
         default:
             name: Display
-            description: Connect a signal to show it on the display - It can be a shape,
-                color or boolean.
+            description: Collega un segnale per mostrarlo sul display.
```

Comment:
`Display` upper or lower case? compare to line 701

Reply:
it's consisted with english, I can change it either way

Reply:
I don't think it is intended. Just change it how nouns are written in Italian

Reply:
But consistently through the whole text if possible 😅

Reply:
well display is a common name so lowercase

Reply:
i think those are the only two instances
i'll do a final pass on everything anyway